features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/data/AuthManager.js

```diff
@@ -163,14 +161,26 @@ class AuthManager {
             validity_period: 3600,
             scopes: 'apim:subscribe apim:signup apim:workflow_approve'
         };
-        let promised_response = axios.post(this.getTokenEndpoint(), qs.stringify(data), {headers: headers});
+        let promised_response = axios(Utils.getLoginTokenPath(), {
+            method: "POST",
+            data: qs.stringify(data),
+            headers: headers,
+            withCredentials: true
+        });
+        //Set the environment that user tried to authenticate
+        let previous_environment = Utils.getEnvironment();
+        Utils.setEnvironment(environment);
+
         promised_response.then(response => {
             const validityPeriod = response.data.validityPeriod; // In seconds
             const WSO2_AM_TOKEN_1 = response.data.partialToken;
-            const user = new User(response.data.authUser, response.data.idToken);
-            user.setPartialToken(WSO2_AM_TOKEN_1, validityPeriod, "/store");
+            const user = new User(Utils.getEnvironment().label, response.data.authUser, response.data.idToken);
+            user.setPartialToken(WSO2_AM_TOKEN_1, validityPeriod, Utils.CONST.CONTEXT_PATH);
             user.scopes = response.data.scopes.split(" ");
             AuthManager.setUser(user);
+        }).catch(error => {
+            console.log("Authentication Error:\n", error);
```

Comment:
Remove console logs or change it to console.error()

Reply:
Changed to `console.error()` for both store and publisher with the commit https://github.com/wso2/carbon-apimgt/pull/4843/commits/c702d08e2742213b1108f09894f93f8235b3c393